docs/conf/australia/2021/news/welcome.rst

```diff
@@ -0,0 +1,66 @@
+:template: {{year}}/generic.html
+
+.. post:: March 15, 2021
+   :tags: {{shortcode}}-{{year}}, website, tickets
+
+Announcing Write the Docs Australia & India {{year}}
```

Comment:
I'll let Swapnil update this with the dates for CFP and tickets, in collab with Sam of course :)